src/rulesets/oas/__tests__/parameter-description.ts

```diff
@@ -78,7 +78,26 @@ describe('parameter-description', () => {
         },
       },
     });
-    expect(results).toMatchSnapshot();
+    expect(results).toEqual([
+      {
+        code: 'parameter-description',
```

Comment:
This is great! If you wanted to make this a bit lighter to read (and going back to the conversation about only asserting the important and relevant bits) then you could do:

expect(results).toEqual([expects.objectContaining({

then keep the code, message, path, etc and maybe kill the range and anything else you don’t really care about. Not a blocked and if you decide you care about them all them keep them all, just letting you know it’s a thing. I was so happy when jakub showed me this 😬

Reply:
Sure, will do. The reason I didn't remove the `range` is that I think it's all important data to check (e.g. someone may rely on this information).

Reply:
Right thats what im saying, include the things you want, if you want specifically range then put that in the spec title, right? The test is asserting that when a problem happens, it will contain a range with these values. If thats what you want then thats the test. Anything you dont care about, get it outta there. :)